class_room.rb

```diff
@@ -0,0 +1,13 @@
+class ClassRoom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students.push(student)
+    student.class_room = self
+  end
```

Comment:
- [ ] Please use `@students` instead of `students` as now students is `attr_reader` and you cannot modify it.
- [ ] Please change the `student.class_room` to `student.classroom` as you name writer that way inside `student` class.